### PR TITLE
feat-badge

### PR DESCRIPTION
**Pull request by iamsaurav28**
Screenshot (65)

### CODE REVIEW

File: stylesheets/components/badge.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
+@import url(/stylesheets/abstract/variables.css);
+@import url(/stylesheets/abstract/color.css);
+@import url(/Stylesheets/main.css);
+
+.mark {
+  margin: 2rem;
+}
+
+.bedge {
```
Comment (krishanu2911) [inline stylesheets/components/badge.css:9]: just check does it is typo?

File: comp-layouts/badge.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+     <link rel="stylesheet" href="/Stylesheets/components/badge.css">
+     <link rel="stylesheet" href="/Stylesheets/main.css">
+     <title>badge</title>
+</head>
+<body>
+
+         <div class="container">
+          <img class="bedge mark"  src="https://img.icons8.com/ios-filled/50/000000/apple-mail.png"/>
+          <img  class="bedge mark" src="https://img.icons8.com/external-flatart-icons-outline-flatarticons/64/000000/external-cart-ecommerce-flatart-icons-outline-flatarticons.png"/>
```
Comment (shubhamkumar648) [inline comp-layouts/badge.html]: add alt atttributes for all image tag

File: stylesheets/components/badge.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
+@import url(/stylesheets/abstract/variables.css);
```
Comment (shubhamkumar648) [inline stylesheets/components/badge.css:1]: put these file in main css

**Review by shubhamkumar648 — COMMENTED** [review]
consider my comment as changes

File: stylesheets/components/badge.css [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+  align-items: center;
+  color: var(--white);
+  height: 2rem;
+  width: 2rem;
+  background-color: var(--red-alert);
+  border-radius: 50%;
+  position: absolute;
+}
+
+.bot {
```
Comment (krishanu2911) [inline stylesheets/components/badge.css:28]: just a suggestion if comfortable try to give common classes if possible

**Review by krishanu2911 — COMMENTED** [review]
Well written code Just Added few suggestion

File: stylesheets/components/badge.css (hunk 3 above)
Comment (iamsaurav28) [reply stylesheets/components/badge.css:1]: its been already addded

File: stylesheets/components/badge.css (hunk 1 above)
Comment (iamsaurav28) [reply stylesheets/components/badge.css:9]: no its not typo i named it bedge only

File: comp-layouts/badge.html (hunk 2 above)
Comment (iamsaurav28) [reply comp-layouts/badge.html]: done

File: stylesheets/components/badge.css (hunk 4 above)
Comment (iamsaurav28) [reply stylesheets/components/badge.css:28]: yes, i tried but for that i have to change other many things, so i didnt made that one common.

Comment (krishanu2911) [reply stylesheets/components/badge.css:28]: ok np

File: stylesheets/components/badge.css (hunk 3 above)
Comment (shubhamkumar648) [reply stylesheets/components/badge.css:1]: okay

**Review by krishanu2911 — APPROVED** [review]

File: stylesheets/components/badge.css (hunk 3 above)
Comment (shubhamkumar648) [reply stylesheets/components/badge.css:1]: Make sure all css file in main.css file

File: comp-layouts/badge.html (hunk 2 above)
Comment (shubhamkumar648) [reply comp-layouts/badge.html]: perfect

**Review by shubhamkumar648 — APPROVED** [review]
consider my comment  as suggetion

File: stylesheets/components/badge.css (hunk 3 above)
Comment (iamsaurav28) [reply stylesheets/components/badge.css:1]: yes i ll

**Pull request merged**